wolfcrypt/src/rsa.c

```diff
@@ -1398,13 +1398,13 @@ static int RsaUnPad(const byte *pkcsBlock, unsigned int pkcsBlockLen,
 #ifndef WOLFSSL_RSA_VERIFY_ONLY
     else {
         word32 j;
```

Comment:
Change i and j to word16 for consistency.

Reply:
Fixed

Reply:
Thanks Kojo!